dart/lib/src/sentry_options.dart

```diff
@@ -276,7 +276,7 @@ class SentryOptions {
   /// breadcrumbs.
   /// In a Flutter environment, this setting also toggles recording of `debugPrint` calls.
   /// `debugPrint` calls are only recorded in release builds, though.
-  bool enablePrintBreadcrumbs = true;
+  bool enablePrintBreadcrumbs = false;
```

Comment:
Is this change needed to be part of the web support? it could have side effects of itself worth tracking as a separate change probably done before the big web support.

Reply:
this is temporary for testing, on web with the js sdk integration this seems to run into a recursion so I've disabled it for now.